Unreal/SystemLocalized/det/Bluff.det

```diff
@@ -124,19 +124,19 @@ Message="Versand-Log: Grorq vom Stamm der Roten Hände. Zum nächsten Vollmond e
 
 [TranslatorEvent25]
 ; EN: Message="Jailer Hrang of the Red Hand Tribe: we have captured a Terran girl. Some of my soldiers want to try and take liberties with her. I guess that's OK as long as they watch those boots! She kicked me in the Hrangos last time."
-Message="Wärter Hrang vom Stamm der Roten Hände: Wir haben ein Weib vom Planeten Terra gefangengenommen. Einige meiner Soldaten wollen sich ein bisschen mit ihr begnügen. Müsste passen, solang sie sich vor diesen Stiefeln in Acht nehmen!. Letztes Mal hat sie mich am kleinen Hrango erwischt."
+Message="Wärter Hrang vom Stamm der Roten Hände: Wir haben ein Weib vom Planeten Terra gefangengenommen. Einige meiner Soldaten wollen sich ein bisschen mit ihr begnügen. Müsste passen, solang sie sich vor diesen Stiefeln in Acht nehmen! Letztes Mal hat sie mich genau in die Hrangos erwischt."
```

Comment:
Message="Wärter Hrang vom Stamm der Roten Hände: Wir haben ein Weib vom Planeten Terra gefangengenommen. Einige meiner Soldaten wollen sich ein bisschen mit ihr vergnügen. Müsste passen, solang sie sich vor diesen Stiefeln in Acht nehmen! Letztes Mal hat sie mich genau in die Hrangos erwischt."